app/src/main/java/org/listenbrainz/android/ui/screens/profile/createdforyou/PlaylistTrackComposable.kt

```diff
@@ -0,0 +1,365 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.shape.GenericShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.FilterQuality
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.unit.times
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import org.listenbrainz.android.R
+import org.listenbrainz.android.model.playlist.PlaylistArtist
+import org.listenbrainz.android.ui.theme.ListenBrainzTheme
+
+/**
+ * A composable that displays a track in a playlist.
+ * @param modifier Modifier to be applied to the composable.
+ * @param trackName Name of the track.
+ * @param durationInSeconds Duration of the track in seconds.
+ * @param artists List of artists who contributed to the track.
+ * @param coverArtUrl URL of the cover art of the track.
+ * @param errorTrackImage Drawable resource to be used in case the cover art is not available.
+ * @param onDropdownIconClick Lambda to be executed when the dropdown icon is clicked.
+ * @param dropDown Composable to be displayed when the dropdown icon is clicked.
+ * @param isReorderButtonVisible Boolean to determine if the reorder button should be visible.
+ * @param color Color of the composable.
+ * @param titleColor Color of the title of the track.
+ * @param subtitleColor Color of the subtitle of the track.
+ * @param goToArtistPage Lambda to be executed when an artist is clicked.
+ * @param onClick Lambda to be executed when the composable is clicked.
+ * @param onPlayClick Lambda to be executed when the play button is clicked.
+ * @param onReorderClick Lambda to be executed when the reorder button is clicked.
+ */
+@Composable
+fun PlaylistTrackComposable(
```

Comment:
Please use `ListenCardSmallDefault` instead of creating new cards, it is very hard to maintain duplicates + consistency is sacrificed. What every extra options you many want to show, show it in `trailingContent` composable lambda.

Also, please do not add the play icon, rather play the song when card click, i.e., when `onClick` is invoked.